validator/client/iface/validator_client.go

```diff
@@ -128,7 +128,7 @@ type ValidatorClient interface {
 	ValidatorStatus(ctx context.Context, in *ethpb.ValidatorStatusRequest) (*ethpb.ValidatorStatusResponse, error)
 	MultipleValidatorStatus(ctx context.Context, in *ethpb.MultipleValidatorStatusRequest) (*ethpb.MultipleValidatorStatusResponse, error)
 	BeaconBlock(ctx context.Context, in *ethpb.BlockRequest) (*ethpb.GenericBeaconBlock, error)
-	ProposeBeaconBlock(ctx context.Context, in *ethpb.GenericSignedBeaconBlock) (*ethpb.ProposeResponse, error)
+	ProposeBeaconBlock(ctx context.Context, forkVersion string, in *ethpb.GenericSignedBeaconBlock) (*ethpb.ProposeResponse, error)
```

Comment:
fork version is needed now, and this actually matches the spec behavior, we'll need version header as input: https://github.com/ethereum/beacon-APIs/blob/master/apis/beacon/blocks/blocks.v2.yaml#L39-L44

Unfortunately with Fulu, we cannot directly infer the version from `GenericSignedBeaconBlock` type anymore